client/components/mini-site-preview/index.jsx

```diff
@@ -11,7 +11,7 @@ import { invoke } from 'lodash';
  */
 import { loadmShotsPreview } from 'my-sites/importer/site-importer/site-preview-actions';
 
-export class MiniSitePreviewWrapper extends Component {
+export class MiniSitePreview extends Component {
```

Comment:
I'm not sure why we `export` here as well as at the bottom, maybe that's just my ignorance.

Reply:
It's good practice to export the base component... but even better practice to test the component, which I've not done here 😬